src/helpers/queryParsing.ts

```diff
@@ -17,12 +17,14 @@ const polymarketCtfAdapterAddressV2 =
   "0x2f5e3684cb1f318ec51b00edba38d79ac2c0aa9d";
 const polymarketCtfExchangeAddress =
   "0x4bFb41d5B3570DeFd03C39a9A4D8dE6Bd8B8982E";
+const polymarketSportsAddress = "0xbec046Ae23FEE91643a45b03f12c6afDb2628d9c";
 
 export const polymarketRequesters = [
   polymarketBinaryAdapterAddress.toLowerCase(),
   polymarketCtfAdapterAddress.toLowerCase(),
   polymarketCtfAdapterAddressV2.toLowerCase(),
   polymarketCtfExchangeAddress.toLowerCase(),
+  polymarketSportsAddress.toLocaleLowerCase(),
```

Comment:
any reason to use `toLocaleLowerCase` ? I'm not certain it'll make any difference with hex

Reply:
oh true, didint notice that.  must have been autocomplete